p2panda-rs/Cargo.toml

```diff
@@ -75,6 +79,7 @@ pretty_env_logger = "^0.4.0"
 proptest = { version = "^1.0.0", default-features = false, features = ["std"] }
 rstest = "^0.15.0"
 rstest_reuse = "^0.3.0"
+tokio = { version = "1.19.2", features = ["rt", "macros", "rt-multi-thread"] }
```

Comment:
Do we have to mention `tokio` again as a dev dependency? I think if its already a dependency it will also be accessible for the `tests` target or is there something special when it is `optional`?

Reply:
I think we do as it's optional in `[dependencies]` which means it wouldn't be included for the `tests` target if we didn't include it again here. Maybe we can do it differently though, like how we are doing for `testing`.

Reply:
Okay, that makes sense, let's keep it like this then :+1: